src/components/card/card.jsx

```diff
@@ -79,6 +85,7 @@ export default class Card extends React.Component {
 
 Card.propTypes = {
   title: PropTypes.string.isRequired,
+  groupName: PropTypes.string,
```

Comment:
perhaps this could be made more generic, like `subTitle`?  This way the `Card` component maintains a generic interface / API.